test/org/loklak/tools/storage/UTF8Test.java

```diff
@@ -0,0 +1,50 @@
+package org.loklak.tools.storage;
+
+import org.junit.Assert;
```

Comment:
import ```assertEquals``` and ```assertArrayEquals``` individually and call them explicitly.

Reply:
Okay Sir... :+1: